README.md

```diff
@@ -319,6 +319,28 @@ evaluation.run(
 <details>
   <summary>  Saving retrieval task predictions </summary>
 
+### Using Late Interaction models for retrieval
+
```

Comment:
Missed this but not a blocker - this could be in its own expand element.

Reply:
https://github.com/embeddings-benchmark/mteb/pull/1592 
Resolved this issue. Apologies for the inconvenience, and thank you very much for your support!